src/integration-test/java/org/ow2/proactive/catalog/service/CatalogObjectServiceIntegrationTest.java

```diff
@@ -417,6 +417,45 @@ public void testCatalogObjectWithProjectNameAsQueryParamAndInMetadataListAndInWo
 
     }
 
+    @Test
+    public void testRestoreCatalogObject() throws IOException, InterruptedException {
+        String workflowName = "workflow";
+        String bucketName = bucket.getName();
+        String updatedProjectName = "updatedProjectName";
+
+        //projectName initialization
+        CatalogObjectMetadata catalogObjectMetadata = catalogObjectService.createCatalogObject(bucketName,
+                                                                                               workflowName,
+                                                                                               PROJECT_NAME,
+                                                                                               "workflow",
```

Comment:
use the variable workflowName instead of "workflow"

Reply:
Why should I use workflowName as a kind ?